test/e2e_node/system_node_critical_test.go

```diff
@@ -100,7 +100,7 @@ var _ = SIGDescribe("SystemNodeCriticalPod", framework.WithSlow(), framework.Wit
 					if err == nil {
 						framework.Logf("mirror pod %q is running", mirrorPodName)
 					} else {
-						framework.Logf(err.Error())
+						framework.Logf("%s", err.Error())
```

Comment:
Side node: I am not sure why we have `framework.Logf` at all.

Our testing guidelines suggest to use it for output from E2E tests, but don't say why it's better than normal klog calls.

Reply:
Oh dear. I added that text and the corresponding feature (= skipping based on GinkgoHelper) and then promptly forgot about it again.

So there is indeed a justification now. But it's recent (6 months ago). Before that, there wasn't really a reason and it's still unclear to me why `framework.Logf` as added initially.

Reply:
check the git  history or ask the test team ?

Reply:
> check the git history 

As it's unfortunately much too common, the commit message which adds the function only explains the "what", but not the "why":
https://github.com/kubernetes/kubernetes/commit/251bb585bd3214a3f5e21ba1b724a46610a5568c

> ask the test team

The team from that time or today's team? The original authors are not active anymore and I don't want to bother people about work they did 10 years ago. Today's team is me and a few others who weren't around at that time.